### PR TITLE
Fix oscap content path and SCA failures

### PR DESCRIPTION
**Pull request by Gauravtalreja1**
Description:
1. Fix oscap content path
2. Adjust  `setup_org_for_a_custom_repo` and `setup_org_for_a_rh_repo` CLI factory helpers for SCA

### CODE REVIEW

**Comment by Gauravtalreja1** [comment]
trigger: test-robottelo
pytest: tests/foreman/ --component SCAPPlugin

File: robottelo/cli/factory.py [hunk 1]
```diff
@@ -1717,14 +1717,19 @@ def setup_org_for_a_custom_repo(options=None):
...
-        {
-            'activationkey-id': activationkey_id,
-            'organization-id': org_id,
-            'subscription': custom_product['name'],
-        }
-    )
+
+    # Add custom_product subscription to activation-key, if SCA mode is disabled
+    if (
+        Org.info({'fields': 'Simple content access', 'id': org_id})['simple-content-access']
```
Comment (jameerpathan111) [inline robottelo/cli/factory.py:1723]: @Gauravtalreja1  good idea to create a helper function for checking if org has sca enabled or not?

Comment (Gauravtalreja1) [reply robottelo/cli/factory.py:1723]: @jameerpathan111 Thanks for the suggestion, updated

**Comment by Gauravtalreja1** [comment]
trigger: test-robottelo
pytest: tests/foreman/ --component SCAPPlugin

**Review by jameerpathan111 — APPROVED** [review]

**Review by omkarkhatavkar — APPROVED** [review]

**Pull request merged**